blackbox/docs/README.rst

```diff
@@ -0,0 +1,71 @@
+====================
+Documentation README
+====================
+
+Top-Level Structure
+===================
+
+The documentation is structured as follows:
+
++------------------------+----------------------------------------------------+
+| Section Title          |  Description                                       |
++========================+====================================================+
+| Release Notes          | A collection of release notes, one document per    |
+|                        | release. Each document should contain upgrade      |
+|                        | information as well as a list of changes.          |
++------------------------+----------------------------------------------------+
+| Installation           | THIS SECTION IS CHANGING SOON                      |
++------------------------+----------------------------------------------------+
+| Configuration          | Documentation for configuration settings, split    |
+|                        | into sections by theme.                            |
++------------------------+----------------------------------------------------+
+| Command Line Interface | Documentation for the ``crate`` command.           |
++------------------------+----------------------------------------------------+
+| CrateDB SQL            | THIS SECTION IS CHANGING SOON                      |
```

Comment:
Looking at this we might need to expand at least this section to the next level.

Reply:
@joemoe this will be fixed by my subsequent work to reorg the docs. so I want to leave this as it is for now